sharepoint/sharepoint-ps/sharepoint-online/Start-SPOAuditDataCollectionForActivityInsights.md

```diff
@@ -0,0 +1,79 @@
+﻿---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spoauditdatacollectionforactivityinsights
+applicable: SharePoint Online
+title: Start-SPOAuditDataCollectionForActivityInsights
+schema: 2.0.0
+author: pvrk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Start-SPOAuditDataCollectionForActivityInsights
+
+## SYNOPSIS
+
+This cmdlet starts collecting relevant audit data for reports, based on activities within the last 28 days, such as Sharing link reports, content shared with Everyone except external users etc from SharePoint admin center.
+
+## SYNTAX
+
+```
+Start-SPOAuditDataCollectionForActivityInsights -ReportEntity <OptInReportEntityEnum> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet starts collecting relevant audit data for reports, based on activites related to sharing and access. Reports are available for the following scenarios:
+
+- Sharing links created in last 28 days (Anyone, People-in-your-org, Specific people shared externally).
```

Comment:
>Anyone, People-in-your-org, Specific people shared externally [](http://example.com/codeflow?start=41&length=61)

are these the "official" names for these -- including casing?

Reply:
[MS public document](https://support.microsoft.com/en-us/office/share-sharepoint-files-or-folders-1fe37332-0f9a-4719-970e-d2578da4941c) regarding sharing links

"Anyone"
"People in <Your Organization>"
"People you choose" - under the heading "Share with specific people" - This product specifically focuses on 'external' people

Reply:
let's use the same wording as them -- eg should be "People in your organization"

@pvrk